.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Add the story URL here. Prefer the short link format, e.g. https://app.clubhouse.io/acme/story/1234/
```

Comment:
It would be better to change this to map to the Github project: 

```suggestion
https://github.com/longnd/elixir-gscraper/projects/
```

Reply:
that makes sense. The change was made in 29970bd.